client/my-sites/plan-features/index.jsx

```diff
@@ -952,9 +954,16 @@ const ConnectedPlanFeatures = connect(
 		const canPurchase = ! isPaid || isCurrentUserCurrentPlanOwner( state, selectedSiteId );
 		const isLoggedInMonthlyPricing =
 			! isInSignup && ! isJetpack && kindOfPlanTypeSelector === 'interval';
+		const isInHideECommerceExperiment =
+			isInSignup && isLaunchPage && 'treatment' === experimentVariation;
```

Comment:
I think `'treatment' === experimentVariation` should suffice since treatment is assigned only when the previous two conditions are met?